src/monegi/debug/assert.c

```diff
@@ -1,5 +1,10 @@
 #include "PR/os_internal.h"
 
+extern void __assertBreak(void);
+#ifndef _FINALROM
+
 void __assert(const char* exp, const char* filename, int line) {
     osSyncPrintf("\nASSERTION FAULT: %s, %d: \"%s\"\n", filename, line, exp);
+	__assertBreak();
```

Comment:
mixing spaces and tabs 💀

Reply:
Fixed in  https://github.com/ModernN64SDK/libultra_modern/pull/3/commits/88193bcfaa58475449a03a81c1ed0d015b70ffe8